### ISSUE TITLE
RHEL-07-040180 not being bypassed

### ISSUE THREAD

**Issue by average-joe** [MindPointGroup/RHEL7-STIG#38]
While running this STIG, I keep getting an error.  See below

`TASK [MindPointGroup.RHEL7-STIG : MEDIUM | RHEL-07-040180 | PATCH | The operating system must implement cryptography to protect the integrity of Lightweight Directory Access Protocol (LDAP) authentication communications.] ***************
fatal: [lab-redjack-01.rogue]: FAILED! => {"failed": true, "msg": "The conditional check ''yes' in rhel_07_040180_audit.stdout' failed. The error was: error while evaluating conditional ('yes' in rhel_07_040180_audit.stdout): Unable to look up a name or access an attribute in template string ({% if 'yes' in rhel_07_040180_audit.stdout %} True {% else %} False {% endif %}).\nMake sure your variable name does not contain invalid characters like '-': argument of type 'StrictUndefined' is not iterable\n\nThe error appears to have been in '/cyclops-ansible/roles/MindPointGroup.RHEL7-STIG/tasks/fix-cat2.yml': line 1805, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"MEDIUM | RHEL-07-040180 | PATCH | The operating system must implement cryptography to protect the integrity of Lightweight Directory Access Protocol (LDAP) authentication communications.\"\n  ^ here\n"}
`

It seems to be having an issue with command: `- "'yes' in rhel_07_040180_audit.stdout"` within:

```
- name: "MEDIUM | RHEL-07-040180 | PATCH | The operating system must implement cryptography to protect the integrity of Lightweight Directory Access Protocol (LDAP) authentication communications."
  command: "true"
  changed_when: no
  when:
      - "'yes' in rhel_07_040180_audit.stdout"
      - rhel_07_040180
  tags:
      - cat2
      - medium
      - patch
      - RHEL-07-040180
      - notimplemented
```

I'm running this on a CentOS 7 server with Ansible 2.4.0.0.

**Comment by dirtyharrycallahan** [comment]
I reversed the when clauses to avoid the test for "yes" unless rhel_07_040180 is selected

**Comment by shepdelacreme** [comment]
Created a PR for a fix in #39 

I'll merge if it looks gtg.

**Comment by shepdelacreme** [comment]
Fixed with #39
